core/src/main/java/com/linecorp/armeria/common/HttpHeaderNames.java

```diff
@@ -232,6 +233,11 @@ public final class HttpHeaderNames {
      */
     public static final AsciiString FOLLOW_ONLY_WHEN_PRERENDER_SHOWN =
             create("Follow-Only-When-Prerender-Shown");
+    /**
+     * The HTTP {@code "Git-Protocol"} header field name, as described in
+     * <a href="https://git-scm.com/docs/protocol-v2#_http_transport">HTTP Transport</a>.
+     */
+    public static final AsciiString GIT_PROTOCOL = create("Git-Protocol");
```

Comment:
Isn't this header too specific to Git protocol and not really a part of the HTTP specification?

Reply:
I thought this would be useful for users who want to implement a Git HTTP server like ours. 🤔 
However, I am open to reverting it if you think it's unnecessary.

Reply:
Let me add `@UnstableApi` and remove them if necessary.